### DISCUSSION TITLE
How to hide dashboards

### DISCUSSION THREAD

**Discussion by weljajoh** [teslamate-org/teslamate#3580] [Q&A] [answered]
Is there a way to hide dashboards? Teslamate now has some new LFP battery related dashboards that I'd like to hide (or delete) since my car doesn't have such a battery. I tried deleting them via Home > Dashboards > Teslamate but got the message that 'provisioned dashboards cannot be deleted'.

**Comment by cwanja** [comment, marked as answer]
We have a feature request / discussion to evaluate exactly that #3311.
